backend/game/engine/player_queue.py

```diff
@@ -1,25 +1,21 @@
 import itertools
 
+from game.player.player import Player
+
 
 class GamePlayers():
 
     def __init__(self):
         self.all_players = []
```

Comment:
`self.all_players` makes it seem like you have other lists which do not contain all the players. This could probably be shortened to just `self.players`